src/pages/login.js

```diff
@@ -65,13 +65,8 @@ const LoginPage = ({ classes = {}, UI = {}, history, loginUserAction }) => {
   );
 };
 
-const mapStateToProps = state => {
-  const user = state.user;
-  const UI = state.UI;
-  return {
-    user,
-    UI,
-  };
+const mapStateToProps = ({ UI }) => {
+  return { UI };
```

Comment:
can be simplified to 
```
=> ({ UI })

Reply:
Fixed. Also, found various other components that were also simplified with the same process.